src/components/Decoder.tsx

```
@@ -131,9 +135,25 @@ export default function Decoder() {
+							const {
+								topics: [topic0],
+							} = fullAbi.encode(abiObj, []);
+							return fullAbi.decode(abiObj, {
+								data: hexTyped,
+								topics: [topic0, ...extras],
+							});
+						})()
+					: fullAbi.decode(abiObj, hexTyped);
 
-			setDecoded(values);
+			setDecoded(values as any);
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)